src/components/auth/node.tsx

```diff
@@ -65,13 +65,40 @@ export function Node({
         )
 
       case 'checkbox':
-      case 'button':
-        triggerSentry({
-          message: `kratos: tried to render input node which is not supported atm: ${attributes.type}`,
-        })
-        return null
+      case 'button': {
+        const label = node.meta.label?.id
+          ? getKratosMessageString(
+              node.meta.label.id,
+              strings.auth.messages,
+              'Undefined'
+            )
+          : 'Could not find translation'
+        return (
+          <div className="mt-10">
+            <hr />
+            <button
+              className="text-xl serlo-button-blue block w-full py-2 mt-10"
+              name={attributes.name}
+              onClick={(e) => {
+                void onSubmit(e, (attributes as { value: string }).value)
+              }}
+              value={(attributes.value as string) || ''}
+              disabled={attributes.disabled || disabled}
+            >
+              {isLoading ? (
```

Comment:
`isLoading` currently results in both buttons showing the animation which is a bit confusing.
Not sure what's the best way to solve that.